astro.config.mjs

```diff
@@ -2,6 +2,7 @@
 import { defineConfig } from "astro/config";
 
 import sitemap from "@astrojs/sitemap";
+import lottie from "astro-integration-lottie";
```

Comment:
https://github.com/giuseppelt/astro-lottie seemed like the easiest way to go :shrug: